### PR TITLE
Fixed csr access test

### PR DESCRIPTION
**Pull request by silabs-hfegran**
* Updated misa/mtvec,
* Removed trigger registers (In line with changes on 40s)

### CODE REVIEW

File: tests/programs/custom/cv32e40x_csr_access_test/cv32e40x_csr_access_test.S
Comment (silabs-robin) [inline tests/programs/custom/cv32e40x_csr_access_test/cv32e40x_csr_access_test.S:1]: > Fixed csr access test 
> Updated misa/mtvec,
> Removed trigger registers (In line with changes on 40s)

Does this mean the test was regenerated from a Yaml file with the scripts?

Comment (silabs-hfegran) [reply tests/programs/custom/cv32e40x_csr_access_test/cv32e40x_csr_access_test.S:1]: No, in this case it was easier to modify it manually

Comment (silabs-hfegran) [reply tests/programs/custom/cv32e40x_csr_access_test/cv32e40x_csr_access_test.S:1]: the mtvec section was copied from the 40s test (same properties), the rest manually written

Comment (silabs-robin) [reply tests/programs/custom/cv32e40x_csr_access_test/cv32e40x_csr_access_test.S:1]: Ok. Understood.

**Pull request merged**